modules/color-picker/react/lib/ColorInput.tsx

```diff
@@ -44,23 +49,36 @@ const CustomHexInput = styled(TextInput)<Pick<ColorInputProps, 'disabled' | 'gro
     },
   ({disabled}) => ({
     backgroundColor: disabled ? colors.soap200 : '',
+  }),
+  ({theme}) => ({
+    paddingLeft: theme.direction === ContentDirection.LTR ? '46px' : 'calc(100% - 86px) /* @noflip */' ,
```

Comment:
Can you add a comment here that this is because the input is always forced to LTR? I would say it's good practice to always explain when/why we need to override `rtl-css-js` (whether it's with `@noflip`, or just based on `theme.direction`).